src/resources/js/components/NotAuthHome.vue

```diff
@@ -0,0 +1,55 @@
+<template>
+<v-list
+        nav
+        dense
+    >
+        <v-list-item
+            v-for="(route, i) in routes"
+            :key="i"
+            :href="route.path"
+        >
+            <v-list-item-icon>
+                <v-icon>{{ route.icon }}</v-icon>
+            </v-list-item-icon>
+            <v-list-item-content>
+                <v-list-item-title v-text="route.name"></v-list-item-title>
+            </v-list-item-content>
+        </v-list-item>
+    </v-list>
+</template>
+
+<script>
+export default {
+  props: ['parentRefs'],
+  data() {
+    return {
+      routes: [
+
+        {
+          path: '/login',
+          name: 'Connexion',
+          icon: 'mdi-account-circle-outline',
+        },
+        {
+          path: '/creer-compte',
+          name: 'Création de compte',
+          icon: 'mdi-account-circle-outline',
+        },
+        {
+          path: '/reset',
+          name: 'Mot de passe oublié',
+          icon: 'mdi-account-circle-outline',
+        },
+        {
+          path: '/reset-password/:token',
+          name: 'Redéfinir mot de passe',
+          icon: 'mdi-account-circle-outline',
+        },
+      ],
+    };
+  },
+  methods: {
+
+  },
+};
+</script>
```

Comment:
why not simply add an array nonAuthRoutes allongSide of the routes array in the sideBar instead of creating a new component with the exact same template as sidebar?
or by adding an attribute ex: `auth_link` to each route to check if the route should be shown or not

Reply:
I deleted this and remade the routes, guard route and side bar to be better 9ac7672